src/components/Cards/CardService.css

```diff
@@ -0,0 +1,11 @@
+.ph-card-service {
```

Comment:
why are you using css, while we chose tailwind?

Reply:
I used some short normal css to include an exact component look like figma design.